Makefile

```diff
@@ -121,9 +127,11 @@ install-precommit-hook: install-pre-commit
 	pre-commit install
 
 test:  ## run backend tests
-# always use settings.ci-test django settings file when running the tests
-# if we use settings.dev it's very possible that some fail just based on the settings alone
-	$(call run_engine_docker_command,pytest --ds=settings.ci-test)
+	$(call run_backend_tests)
+
+test-dev:  ## very similar to `test` command, but allows you to pass arbitray args to pytest
+           ## for example, `make test-dev ARGS="--last-failed --pdb"
+	$(call run_backend_tests,$(ARGS))
```

Comment:
I wanted to be able to do something like this, without having to manually modify this file:
```bash
$ make test-dev ARGS="apps/api/tests/test_shift_swaps.py --last-failed --pdb"
```